modules/transformations/snowflake/test/integration/ST_BUFFER.test.js

```diff
@@ -0,0 +1,20 @@
+const { runQuery } = require('../../../../../common/snowflake/test-utils');
+
+const pointFixturesIn = require('./buffer_fixtures/in/point');
+const pointFixturesOut = require('./buffer_fixtures/out/point');
+
+test('ST_BUFFER should work', async () => {
+    const featureJSON = JSON.stringify(pointFixturesIn.geom.geometry);
+    const query = `SELECT ST_ASTEXT(ST_BUFFER(TO_GEOGRAPHY('${featureJSON}'), 1000)) as buffer;`;
+    const rows = await runQuery(query);
+    expect(rows.length).toEqual(1);
+    expect(rows[0].BUFFER).toEqual(pointFixturesOut.value);
+});
+
+test('ST_BUFFER should work with segments', async () => {
+    const featureJSON = JSON.stringify(pointFixturesIn.geom.geometry);
+    const query = `SELECT ST_ASTEXT(ST_BUFFER(TO_GEOGRAPHY('${featureJSON}'), 1000, 8)) as buffer;`;
+    const rows = await runQuery(query);
+    expect(rows.length).toEqual(1);
+    expect(rows[0].BUFFER).toEqual(pointFixturesOut.value);
+});
```

Comment:
Maybe we can add a test for default values as we do in other turf UDFs.

Reply:
The default is covered. I'll change the test passing the segment to a value different than the default one